Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -251,7 +246,7 @@ class DataDecoderTask
 
     for (auto it = parser.begin(), end = parser.end(); it != end; ++it) {
       // retrieving RDH v4
```

Comment:
The v4 can go away in the comment. In addition I'd say the full comment can go away (the one below about size as well), as the code is descriptive enough without the comment ?

Reply:
I agree, the comments are now removed.